02_configure_host.sh

```diff
@@ -54,6 +54,18 @@ if [[ ! -z "${MIRROR_IMAGES}" || $(env | grep "_LOCAL_IMAGE=")  || ! -z "${ENABL
     setup_local_registry
 fi
 
+# Configure a local proxy to be used for the installation
+if [[ ! -z "${INSTALLER_PROXY}" ]]; then
```

Comment:
nit: -n instead of ! -z
guess it's too late as we use ! -z everywhere!

Reply:
Yeah we should perhaps consider a future PR to align on one convention (and other issues like quoting and `[` vs `[[` - not a blocker for this though IMO since it's a cosmetic (and as you say pre-existing) issue